src/core/filters/filterregistry.js

```diff
@@ -121,21 +121,35 @@ export default class FilterRegistry {
   }
 
   /**
-   * Transforms a {@link SimpleFilterNode} to answers-core's {@link Filter}
+   * Transforms a {@link SimpleFilterNode} to answers-core's {@link Filter} or {@link CombinedFilter}
+   * if there are multiple matchers.
    *
    * @param {SimpleFilterNode} filterNode
    * @returns {Filter}
    */
   _transformSimpleFilterNode (filterNode) {
```

Comment:
Is it still considered a SimpleFilterNode if it has multiple matches? I'm reading the docs and it says (below). Multiple matchers seems to not follow the single value part. The reason I bring this up is because I was going to suggest maybe another function based on number of matchers? It almost seems like we should try to recurse a second time in `_transformFilterNodes` to get the CombinedFilter.

```
 * A SimpleFilterNode represents a single, atomic filter.
 * An atomic filter is a filter that filters by a single value on a single field id,
 * and does not contain any children filters.
```

Reply:
Yup still a SimpleFilterNode! I can update the doc